cs/src/Management/ITunnelManagementClient.cs

```diff
@@ -345,5 +345,18 @@ public interface ITunnelManagementClient : IDisposable
         /// <param name="cancellation">Cancellation token.</param>
         /// <returns>Array of <see cref="ClusterDetails"/></returns>
         Task<ClusterDetails[]> ListClustersAsync(CancellationToken cancellation = default);
+
+        /// <summary>
+        /// Gets the ssh public key for an ssh client to connect to a shared ssh server.
```

Comment:
Please explain better what scenario the key is for and what one would do with the key (add to `authorized_keys`).